feature/google/build.gradle

```diff
@@ -26,4 +26,6 @@ dependencies {
     testImplementation "io.mockk:mockk:$mockkVersion"
     testImplementation "org.assertj:assertj-core:$assertJVersion"
     testImplementation 'com.google.android.gms:play-services-ads-identifier:17.0.1'
+    testUnityIAPImplementation "com.android.billingclient:billing:$billingVersion"
```

Comment:
Is this necessary? It got removed in https://github.com/RevenueCat/purchases-android/pull/566/files#diff-0fbab3ddfeb7446f9301cb2a204297ff9a8e67e8a3a89418ba5a9c78a14d3f16L27 so I just want to make sure this is not a bad merge

Reply:
i purposely added it back, though i can't remember why now.... will retest

Reply:
ok i'm stuck on this one... When I remove it, feature:google:compileUnityIAPDebugUnitTestKotlin [fails](https://app.circleci.com/pipelines/github/RevenueCat/purchases-android/2996/workflows/5c6d1749-d4e2-4170-807b-86dd4dfa5316/jobs/7400) because it can’t find BillingClient classes.

which sounds like it's basically the same as [this](https://github.com/RevenueCat/purchases-android/pull/566/files#r922197170) discussion...but then i don't understand why this worked on main 🤷‍♀️  it made sense to me at one point

Reply:
@vegaro mind reviewing this when you get a chance? merging is blocked

Reply:
I am sorry for the wait @beylmk

If it's giving you issues with missing classes then it's needed so we need the change